ui/apps/platform/cypress/integration/clusters/Clusters.helpers.js

```diff
@@ -54,7 +54,7 @@ const title = 'Clusters';
 // assert
 
 export function assertClusterNameInSidePanel(clusterName) {
```

Comment:
Nit: is it worth renaming this function as well since the side panel is no more?